multisig/simulations/vip-002/vip-002-ethereum/simulations.ts

```diff
@@ -0,0 +1,569 @@
+import { expect } from "chai";
```

Comment:
I would recommend a file path similar to this
multisig/simulations/ethereum/vip-002/simulations-mainnet.ts
multisig/simulations/ethereum/vip-002/simulations-sepolia.ts

VIP 002 already exists on other networks but is different to this VIP so it will get confusing as more and more VIPS are added in different networks with the same number but unrelated changes.

Having the top directory as the chain resolves this. Then you can organize the VIPs by number. The VIP number only needs to be in the file path once. The under the VIP you can organize the main and test net simulations.

Or if we want to keep the current structure all VIPS should be numbered with the BNB chain proposal ID since they will all originate from there

In this case a good file path would be. It is clean and reduces duplicate information
multisig/simulations/vip-002/ethereum.ts

Reply:
We started with this struct because during the initial steps the VIP's should be similar in the different networks:

* 000: accept ownership of VTreasury
* 001: configure oracles
* 002: configure markets/pools
* 003: configure XVS bridge
* 004: configure XVSVault
* 005: configure PSR
* 006: configure RewardsDistributor
* 007: configure Prime
* 008: enable Prime

And we thought it would be easier to have together the VIP's making the same changes. We won't maintain this in the future, of course, because I assume there will be changes to do in one network not needed in the rest of the networks. In that moment, we won't "reuse" the folders.

Given that, I'm open to changing the folder structure, let's agree about it offline and do the needed changes